lib/v2/learnblockchain/posts.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const cid = ctx.params.cid || 'all';
+    const sort = ctx.params.sort;
+
+    let url = 'https://learnblockchain.cn/categories/';
+    url += cid + '/';
+
+    if (sort) {
+        url += sort + '/';
+    }
+
+    const response = await got({
+        url
+    });
+
+    const data = response.data;
+    const $ = cheerio.load(data);
+    const list = $('div.stream-list section.stream-list-item');
+
+    ctx.state.data = {
+        title: `登链社区--${cid}`,
+        link: url,
+        description: `登链社区`,
+        item: list && list.map((idx, ite) => {
+            const item = $(ite);
+            const json = {
+                title: item.find('h2.title').text().trim(),
+                description: item.find('div.excerpt').text().trim(),
+                pubDate: item.find('.author li:nth-child(2)').text().replace('发布于', '').trim(),
```

Comment:
There are some items with `Invalid Date` in `/learnblockchain/posts/all`. You can use `parseRelativeDate()` for that.
docs: https://docs.rsshub.app/joinus/pub-date.html